docs/ethereum/wallet-detection.md

```diff
@@ -19,3 +19,10 @@ const isBraveWallet = await window.ethereum.request({
   })
 console.log('Brave Wallet: ', isBraveWallet)
 ```
+
+
+Simplified detection (currently only available in Brave Browser Beta):
```

Comment:
This is actually already available in Release channel so you can remove the above. Please also add the alternate detection method I mentioned which is a bit updated from the above.